### ISSUE TITLE
Abnormal alveolar volume (VA)

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6499]
**Preferred term label:**
Abnormal alveolar volume (VA)

**Synonyms**

**Definition (free text, please give PubMed ID)**
Alveolar volume (VA) is a volume accessible during 10-second breath-hold, measured during a single breath manoeuvre. VA is calculated by knowing the fractional concentration of the tracer gas (eg helium) and the volume of the gas inhaled. VA = Vi x(Fi tracer/Fa tracer); Vi - inspired volume of tracer gas, Fi tracer- inspired fraction of tracer gas, Fa tracer - alveolar (exhaled) fraction of tracer gas. PMID 28049168

**Parent term (use hpo.jax.org/app)**
Abnormality in pulmonary function testing HP:0030878

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Pleural mesothelioma ORPHA:50251
Pleural empyema ORPHA:449266
Idiopathic pulmonary fibrosis ORPHA:2032
Congenital lobar emphysema ORPHA:1928
Combined pulmonary fibrosis-emphysema syndrome ORPHA:300564

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik could you provide a definition of alveolar volume -- especially, how is it different from other measures of lung volume such as TLC? I have not been able to find a review article on this topic.

**Comment by eswietlik** [comment]
@pnrobinson VA term definition and citation now updated

**Comment by pnrobinson** [comment]
add increased/decreased

**Comment by pnrobinson** [comment]
@eswietlik  adding this comment to provde context
Only a portion of each tidal volume (Vt) reaches the alveoli and participates in gas exchange. This is referred to as the alveolar volume (Va). The remainder, the so-called dead space volume (Vd), consists of gas that fills the upper and lower airways (anatomic dead space) or reaches nonperfused or relatively poorly perfused alveoli (alveolar dead space).
